code/game/atoms.dm

```diff
@@ -14,6 +14,7 @@
 	var/simulated = 1 //filter for actions - used by lighting overlays
 	var/atom_say_verb = "says"
 	var/dont_save = 0 // For atoms that are temporary by necessity - like lighting overlays
+	var/flags_1 = NONE

```

Comment:
Put the ricochet flag into flags_2. We don't need three variable for flags, we already have two